bin/docker-setup.sh

```diff
@@ -73,6 +73,12 @@ cli config set DOCKER_REQUEST_URL "( ! empty( \$_SERVER['HTTPS'] ) ? 'https://'
 cli config set WP_SITEURL DOCKER_REQUEST_URL --raw
 cli config set WP_HOME DOCKER_REQUEST_URL --raw
 
+echo "Enabling WP_DEBUG"
```

Comment:
Slightly misleading, as we enable more flags there. What about "Enabling WordPress debug-flags"?